device-scripts/README.md

```diff
@@ -0,0 +1,47 @@
+# Diagnostic Tool for a Device
+
+This is a diagnostic tool that can be used to help diagnose problems
+with devices. It must be run as root because it needs to access 
+network, and TLS keys and in general secure directories.
+
+## How to Get It
+The script is available in the lmp-tools repo and can be downloaded and setup like:
+```
+wget https://raw.githubusercontent.com/foundriesio/lmp-tools/master/device-scripts/diag.sh
+chmod 755 diag.sh
+sudo -s
+```
```

Comment:
````suggestion

```
wget https://raw.githubusercontent.com/foundriesio/lmp-tools/master/device-scripts/diag.sh
chmod 755 diag.sh
sudo -s
```
````

Fenced code blocks should be surrounded by blank lines